hal/lib/athena/ChipObject.h

```diff
@@ -13,12 +13,6 @@
 #include <stdint.h>
 
 #include "FRC_FPGA_ChipObject/RoboRIO_FRC_ChipObject_Aliases.h"
```

Comment:
That header is ours, actually. It has our copyright and attempts to follow the Google style guide.